core/store/src/flat/store_helper.rs

```diff
@@ -0,0 +1,211 @@
+use crate::flat::delta::{FlatStateDelta, KeyForFlatStateDelta};
```

Comment:
Could you add some short comment describing what is the purpose of this file? Like, "helper functions for accessing flat storage data in DB" and maybe add todo to remove it in the future, as we discussed.

I'm tempted to ask to add short comments in the beginning of each file, because it can make onboarding of new devs easier. But other files (except types.rs) mostly describe one structure each of which already looks overdocumented, and I don't want to duplicate comments.

Reply:
done for store helpers: 93ba5fe2f

I'm planning to add more docs at the final stage of refactor to avoid unnecessary documentation work